icevision/models/mmseg/models/deeplabv3plus/backbones/resnet.py

```diff
@@ -0,0 +1,283 @@
+__all__ = [
+    "resnet101_d8",
+    "resnet101_d16_mg124",
+    "resnet101b_d8",
+    "resnet18_d8",
+    "resnet18b_d8",
+    "resnet50_d8",
+    "resnet50b_d8",
+]
+
+from icevision.imports import *
+from icevision.models.mmseg.utils import *
+
+model_name = "deeplabv3plus"
+
+
+class MMSegDeeplabPlusBackboneConfig(MMSegBackboneConfig):
+    def __init__(self, **kwargs):
+        super().__init__(model_name=model_name, **kwargs)
+
+
+base_config_path = mmseg_configs_path / f"{model_name}"
+base_weights_url = "https://download.openmmlab.com/mmsegmentation/v0.5/deeplabv3plus"
+
+resnet101_d16_mg124 = MMSegDeeplabPlusBackboneConfig(
+    backbone_type="R101-D16-MG124",
+    pre_trained_variants=[
+        {
+            "config_path": "deeplabv3plus/deeplabv3plus_r101-d16-mg124_512x1024_40k_cityscapes.py",
+            "crop_size": (512, 1024),
+            "lr_schd": "40k",
```

Comment:
what is `lr_schd`?

Reply:
It corresponds to different learning schedules used during the training of pretrained models (see https://github.com/open-mmlab/mmsegmentation/tree/master/configs/_base_/schedules).

This is one of the many complexities of `mmsegmentation` - there are many variations and permutations when it comes to pretrained models: different datasets, different learning schedules, different input size, etc...